src/CloseButton.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { mapToCssModules } from './utils';
+
+const propTypes = {
+  active: PropTypes.bool,
+  'aria-label': PropTypes.string,
+  onClick: PropTypes.func,
+  variant: PropTypes.oneOf(['white', 'black'])
+}
+
+const defaultProps = {
+  variant: 'black',
```

Comment:
I don't think bootstrap has the `black` variant. This is just the `btn-close`. The only variant we should support is `white`.

Btw, `Button` has a `close` prop right now. We might want to use this `Close` button if that prop is passed.